cli/tests/integration_tests.rs

```diff
@@ -2062,6 +2062,18 @@ fn test_permissions_rw_no_prefix() {
   }
 }
 
+#[test]
+fn test_permissions_read_watch_fs_with_relative_path() {
+  let (_, err) = util::run_and_collect_output(
+    true,
+    "run --allow-read=. complex_permissions_test.ts readWatchFs tls",
+    None,
+    None,
+    true,
+  );
+  assert!(!err.contains(util::PERMISSION_DENIED_PATTERN));
+}
+
```

Comment:
A better way to write this test is:
```rust
#[test]
fn test_permissions_read_watch_fs_with_relative_path() {
  let status = util::deno_cmd()
    .current_dir(&util::tests_path())
    .arg("run")
    .arg("--allow-read=.")
    .arg("complex_permissions_test.ts")
    .arg("readWatchFs")
    .arg("tls")
    .spawn()
    .unwrap()
    .wait()
    .unwrap();
  assert!(status.success());
}
```
It's not necessary to collect output - we just need to check the process didn't fail.